### PR TITLE
Replace Logger declarations

### PR DESCRIPTION
**Pull request by msladek**
https://synjira.atlassian.net/browse/CELDEV-454

### CODE REVIEW

File: component/src/main/java/com/celements/blog/article/Article.java [hunk 1]
```diff
@@ -303,7 +303,7 @@ int getMaxNumChars() {
     }
 
     String extr = extract.get(lang);
-    // mLogger.info("getExtract('" + lang + "', " + isViewtypeFull + "') => '" + extr + "'");
+    // LOGGER.info("getExtract('" + lang + "', " + isViewtypeFull + "') => '" + extr + "'");
```
Comment (fpichler) [inline component/src/main/java/com/celements/blog/article/Article.java]: please remove commented code.

File: component/src/main/java/com/celements/blog/article/ArticleParent.java [hunk 2]
```diff
@@ -34,9 +34,9 @@
         docParents.add(nextParent);
       }
     } catch (QueryException exp) {
```
Comment (fpichler) [inline component/src/main/java/com/celements/blog/article/ArticleParent.java]:
```suggestion
    } catch (QueryException | XWikiException exp) {
```

File: component/src/main/java/com/celements/blog/article/ArticleParent.java [hunk 3]
```diff
@@ -34,9 +34,9 @@
...
-      _LOGGER.error("Failed to get parent reference. ", exp);
+      LOGGER.error("Failed to get parent reference. ", exp);
     } catch (XWikiException exp) {
```
Comment (fpichler) [inline component/src/main/java/com/celements/blog/article/ArticleParent.java]:
```suggestion
```

File: component/src/main/java/com/celements/blog/article/ArticleParent.java [hunk 4]
```diff
@@ -34,9 +34,9 @@
...
-      _LOGGER.error("Failed to get parent reference. ", exp);
+      LOGGER.error("Failed to get parent reference. ", exp);
```
Comment (fpichler) [inline component/src/main/java/com/celements/blog/article/ArticleParent.java:37]:
```suggestion
```

**Review by fpichler — CHANGES_REQUESTED** [review]

**Comment by msladek** [comment]
@fpichler please note that I only performed workspace wide search&replace operations. it isn't the original intention of these PRs to refactor anything else except the logger declarations and definitons.

some formattings may have occured automatically on file save according to our formatting rules.

**Review by fpichler — APPROVED** [review]

**Pull request merged**